### ISSUE TITLE
Any instructions on how to get the Ligatures and Texture healing to work in Visual Studio (Not Code)? 

### ISSUE THREAD

**Issue by MrScottyTay** [githubnext/monaspace#35]
Basically the title, I know this is probably more of a Visual Studio question but since it pertains specifically to this project, especially texture healing I thought it would be best to just ask here.

**Comment by dominicsayers** [comment]
Can confirm the ligatures are not working for me in VS Code on Windows 11. I used the suggested setting:

```
    "editor.fontFamily": "Monaspace",
    "editor.fontLigatures": "'ss01', 'ss02', 'ss03', 'ss04', 'ss05', 'ss06', 'ss07', 'ss08', 'calt', 'dlig'",
```

And I'm getting an unwanted ligature for `fi` but not for coding ligatures such as `!=`.

![image_720](https://github.com/githubnext/monaspace/assets/34939/9efdc25f-7fc6-45c9-99a5-7ea9b66434c2)

**Comment by apkd** [comment]
Similarly, I am unable to get ligatures working in Jetbrains Rider. Looks like the `Enable ligatures` setting only enables texture healing, but doesn't affect code ligatures.

![image](https://github.com/githubnext/monaspace/assets/13803120/e82ead60-d6d6-4465-b3ae-6ada9d8eeae2)

**Comment by OliverRC** [comment]
The README seems to indicate this:
https://github.com/githubnext/monaspace#editors

**Comment by apkd** [comment]
> The README seems to indicate this: https://github.com/githubnext/monaspace#editors

❔

**Comment by Birdie0** [comment]
Set proper font family, it should be `Monascape Argon` (replace `Argon` with `Krypton/Neon/Radon/Xenon` if you want to use them instead), that in the case if you installed fonts from `otf` directory. Fonts from `variable` directory have `Var` suffix in name.

```
// installed from `otf` dir
"editor.fontFamily": "Monaspace Argon"
// installed from `variable` dir
"editor.fontFamily": "Monaspace Argon Var"
```

**Comment by nic547** [comment]
Visual Studio enables* ligatures by default, you can't turn them off.
You just need to select the font and restart VS (ligatures won't work before restarting)

The texture healing seems to work for me (using the variable fonts), you can compare a standalone 'm' to 'imi' to see if it works.
Currently enabling ligatures doesn't seem to enable code ligatures, but that's an issue not limited to VS.

**Comment by dominicsayers** [comment]
I agree that using `"editor.fontFamily": "Monaspace Argon Var"` improves the situation. It's worth saying the current documentation is silent on this point and could be improved.

**Comment by FiniteSingularity** [comment]
I too am having issues with ligatures in Visual Studio in Windows 11.  Have installed the fonts, selected `Monaspace Neon Var`, and restarted Visual Studio.  Texture healing seems to be working, but I'm not seeing any ligatures.

Ligatures do seem to work fine with FiraCode in Visual Studio, and in VSCode with the Monaspace fonts.

One thing to note- the Monaspace fonts are not showing up as monospace in Visual Studio (which bolds monospace fonts in the list). Not sure if I botched the install or this is intended.

**Comment by Aurel-Cros** [comment]
I've been fiddling around and, following the [README instructions](https://github.com/githubnext/monaspace#visual-studio-code) I finally got ligatures to work by using these settings : 
```
    "editor.fontFamily": "Monaspace Krypton",
    "editor.fontLigatures": "'calt', 'liga', 'dlig', 'ss01', 'ss02', 'ss03', 'ss04', 'ss05', 'ss06', 'ss07', 'ss08'",
```
Using `"editor.fontLigatures": true,` wouldn't work for me using either the variable or normal font.

**Comment by jpmnteiro** [comment]
Can confirm @Aurel-Cros solution also worked for VSCode running on Ubuntu 22.04

```
  "editor.fontFamily": "'Monaspace Neon', monospace",
  "editor.fontLigatures": "'calt', 'liga', 'dlig'",
```

Worth noting that the fonts and ligatures worked without a hitch on Jetbrain's IDEs.

**Comment by Truleu** [comment]
This seems to be a solution fo VSCode and that's ok, but what about Visua Studio (not Code)?
Ligatures are not working for me either

> I've been fiddling around and, following the [README instructions](https://github.com/githubnext/monaspace#visual-studio-code) I finally got coding ligatures to work by using these settings :
> 
> ```
>     "editor.fontFamily": "Monaspace Krypton",
>     "editor.fontLigatures": "'calt', 'liga', 'dlig'",
> ```
> 
> Using `"editor.fontLigatures": true,` wouldn't work for me using either the variable or normal font.

**Comment by mark-roberts1** [comment]
Just adding my +1 here - I really like how monaspace neon looks, but ligatures do not work on Visual Studio (not vs code!).

The texture healing feature works well, however.
![image](https://github.com/githubnext/monaspace/assets/2751598/ba79961e-2fd1-44a4-84e5-766f2e2fdd5a)

**Comment by heathercran** [comment]
Release 1.2 includes Frozen versions of the fonts, which include all ligatures and texture healing enabled by default. This should be what you're looking for in Visual Studio!
